test/ui/views/choose_language_view_test.dart

```diff
@@ -0,0 +1,56 @@
+// FLUTTER / DART / THIRD-PARTIES
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+
+// VIEW
+import 'package:notredame/ui/views/choose_language_view.dart';
+
+import '../../helpers.dart';
+
+void main() {
+  AppIntl intl;
+  group('SettingsView - ', () {
+    setUp(() async {
+      intl = await setupAppIntl();
+      setupNavigationServiceMock();
+      setupCacheManagerMock();
+      setupSettingsManagerMock();
+    });
+
+    tearDown(() {});
```

Comment:
I'm not sure, but I think in the tearDown, you need to call `unregister<OneOfTheServiceYouUsed>();` for each service you have setup (I found an example [here](https://github.com/ApplETS/Notre-Dame/blob/master/test/ui/views/login_view_test.dart))